### DISCUSSION TITLE
How to jump to the method definition？

### DISCUSSION THREAD

**Discussion by StoneMa** [codefori/vscode-ibmi#872] [Q&A]
When i use IBM i in vscode, i just find i can't go to the function definition through `ctrl+left click`
![image](https://user-images.githubusercontent.com/6917909/190330299-9ff49663-84b9-46da-a8aa-65a36d1628f6.png)

when i click mouse ,it tells me the file was not found
![image](https://user-images.githubusercontent.com/6917909/190330227-c08813f9-d068-4117-9601-3c675b47d32d.png)

**Comment by worksofliam** [comment]
This is likely because the COBOL extension doesn't know how to access our file system, or something like that. Will see what I can do.

**Reply by StoneMa** [reply]
Okay，Thanks a lot.

**Reply by StoneMa** [reply]
Hi，I think it works，today！
![image](https://user-images.githubusercontent.com/6917909/194687044-74c4edea-f87f-427f-a540-ed323107a02b.png)
![image](https://user-images.githubusercontent.com/6917909/194687054-9bcd5854-79ff-421e-9238-3ee3bf6c6203.png)

**Reply by worksofliam** [reply]
Nice! You can thank the cobol people for that
